woocommerce-pdf-ips-number-tools.php

```diff
@@ -132,7 +132,7 @@ public function number_tools_page( $active_tab = '', $active_section = '' ) {
 			'tools'   => __( 'Tools', 'woocommerce-pdf-ips-number-tools' ),
 		];
 		?>
-		<div class="wcpdf_document_settings_sections">
+		<div class="wcpdf-number-tools-sections">
```

Comment:
following the previous comment, this should get a more generic class

Reply:
Solved in commit: https://github.com/wpovernight/woocommerce-pdf-ips-number-tools/pull/13/commits/54b017a17ffcaf5343a684ee84c4f3a4e3a52dc6